.devcontainer/Dockerfile

```diff
@@ -17,14 +19,18 @@ RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- \
     --default-toolchain ${NIGHTLY_VERSION} -y \
     && $HOME/.cargo/bin/rustup component add rust-src --toolchain ${NIGHTLY_VERSION} \
     && $HOME/.cargo/bin/rustup target add riscv32i-unknown-none-elf \
-    && $HOME/.cargo/bin/cargo install cargo-generate cargo-espflash espmonitor bindgen ldproxy
-ENV ESP_BOARD=esp32c3
-ENV ESP_IDF_VERSION=release/v4.4
+    && $HOME/.cargo/bin/cargo install cargo-espflash espmonitor ldproxy
```

Comment:
I think by removing `bindgen` the code will not compile anymore.

Reply:
I also thought that, but when @ivmarkov raised the question in https://github.com/esp-rs/rust/discussions/121#discussioncomment-2720037 I did some testing and found out that is not necessary. All the projects are building fine.

Reply:
You are right, good catch. :+1: